### ISSUE TITLE
Haptic Feedback

### ISSUE THREAD

**Issue by programminghoch10** [jloehr/HID-Wiimote#33]
How about adding Haptic Feedback?
Might also be called Force Feedback.
Some Steering Wheels Support Force Feedback to let the Wheel vibrate on rough Terrain or during a curve.
Is it possible to implement such behavior to the WiiMote?

**Comment by jloehr** [comment]
Wii Remotes do have a rumble motor. However according to [[1](http://www.microchip.com/forums/FindPost/546202)] Force Feedback is some proprietary Microsoft invention, that's been discontinued long time ago. The post also says that Force Feedback won't work on modern x64 system, as Microsoft does not ship any binaries for it anymore.
It would be quite interesting to reverse engineer some old games and try to get Force Feedback support on modern systems, but that'll be quite some work.
So long story short, no there won't be any Force Feedback support any time soon.
For haptic feeback (as in not Force Feedback), the HID specs do include usage pages for that, but to my knowledge no game utilizes that. Those probably just use the Force Feeback implementation, that is some part of Direct Input.   

[1] - http://www.microchip.com/forums/FindPost/546202
